docs/sources/tempo/traceql/metrics-queries.md

```diff
@@ -151,20 +200,26 @@ To demonstrate this flexibility, consider this nonsensical quantile on `span.htt
 { name = "GET /:endpoint" } | quantile_over_time(span.http.status_code, .99, .9, .5)
 ```
 
+
+
 ### The `compare` function
 
-This adds a new metrics function `compare` which is used to split the stream of spans into two groups: a selection and a baseline.
+The `compare` function is used to split the stream of spans into two groups: a selection and a baseline.
 It returns time-series for all attributes found on the spans to highlight the differences between the two groups.
-This is a powerful function that is best understood by looking at example outputs below:
+This is a powerful function that is best understood by looking at example outputs below.
 
 The function is used like other metrics functions: when it's placed after any search query, and converts it into a metrics query:
 `...any spanset pipeline... | compare({subset filters}, <topN>, <start timestamp>, <end timestamp>)`
 
 Example:
+
 ```
 { resource.service.name="a" && span.http.path="/myapi" } | compare({status=error})
 ```
-This function is generally run as an instant query.  It may return may exceed gRPC payloads when run as a query range.
+
+This function is generally run as an instant query.
```

Comment:
Where is it explained what an instant query is? I see an explanation in the metrics-query.md but maybe we should say something here too, in the context of TraceQL metrics

Reply:
Added.